On 32bit arches size_t may not necessarily be defined as long unsigned int.

diff -upr Bear-3.0.0.orig/source/citnames/source/Configuration.cc Bear-3.0.0/source/citnames/source/Configuration.cc
--- Bear-3.0.0.orig/source/citnames/source/Configuration.cc	2020-10-08 14:18:32.551685952 +0200
+++ Bear-3.0.0/source/citnames/source/Configuration.cc	2020-10-08 14:18:51.952022987 +0200
@@ -207,7 +207,7 @@ namespace cs {
             nlohmann::json out = rhs;
             os << std::setw(4) << out << std::endl;
 
-            return rust::Ok(1ul);
+            return rust::Ok((size_t)1ul);
         } catch (const std::exception &error) {
             return rust::Err(std::runtime_error(error.what()));
         }
